### PR TITLE
Create moving-to-uk.md

### PR DESCRIPTION
**Pull request by sighmin**
A guide for moving to the UK from our collective experience. Similar to [moving to usa](https://github.com/thoughtbot/cities/blob/master/moving-to-usa.md) guide.

Born from this thread in Slack: https://thoughtbot.slack.com/archives/C04RN8H1H/p1542877182127600

### CODE REVIEW

**Comment by sighmin** [comment]
@justmendes Putting this here for reference https://docs.google.com/document/d/16QmyFqxByZ8oOCHI8QCOA70M5BGwV6rX1VyDyW9bDOU/edit

Initial google doc we collaborated on 👆

File: moving-to-uk.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# Moving to UK from overseas
+
+Tips from our collective experience of moving to the UK.
+
+If you're moving to the USA, [look here](/moving-to-usa.md).
+
+* VISAs
```
Comment (nickcharlton) [inline moving-to-uk.md]: `VISA` → `Visa`; I think `VISA` is the card company and `Visa` isn't normally capitalised fully?

**Review by nickcharlton — APPROVED** [review]
nice! This is looking good.

**Comment by sighmin** [comment]
Thanks for the feedback @nickcharlton 🌟

**Comment by sighmin** [comment]
I think this is ready to go. @nickcharlton could you give this a once over on Friday?

**Review by nickcharlton — APPROVED** [review]
Looks good!

Only final thing I'd suggest is to wrap the lines at 80 characters like we do everywhere.

**Pull request merged**